presto-native-execution/presto_cpp/main/tests/TaskManagerTest.cpp

```diff
@@ -11,14 +11,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include "presto_cpp/main/TaskManager.h"
 #include <gtest/gtest.h>
-#include "velox/common/base/tests/Fs.h"
-
 #include "presto_cpp/main/PrestoExchangeSource.h"
-#include "presto_cpp/main/TaskManager.h"
 #include "presto_cpp/main/TaskResource.h"
-#include "presto_cpp/main/common/Configs.h"
 #include "presto_cpp/main/tests/HttpServerWrapper.h"
+#include "velox/common/base/tests/Fs.h"
```

Comment:
Hi @spershin , will leave it to you to pick up the Fs.h path change when advancing the Velox version. If this build fails due to the patch change, here the Fs.h is moved from

velox/common/base/tests/Fs.h

to 

velox/common/base/Fs.h

Reply:
Yes, I'll get this in, at the current version it is not in because I'm looking for the breaking change.